spring-cloud-kubernetes-dependencies/pom.xml

```diff
@@ -246,31 +245,10 @@
 				<scope>test</scope>
 			</dependency>
 
-			<dependency>
-				<groupId>io.fabric8</groupId>
-				<artifactId>mockwebserver</artifactId>
-				<version>${mockwebserver.version}</version>
-				<scope>test</scope>
-				<exclusions>
-					<exclusion>
```

Comment:
Could this have been included due to the exclusions?

Reply:
may at the time it was messing some tests, but not currently. I would have given much more thought time to this (and probably even try to figure out transitive dependencies everywhere), but this has `<scope>test</scope>`, which imho, simplifies the explanation here greatly. wdyt?

Reply:
fair enough, I will merge this one, and the other ones on Monday

Reply:
ok, thank you.